apps/web/src/components/Menu/hooks/useIfoStatus.ts

```diff
@@ -0,0 +1,57 @@
+import useSWRImmutable from 'swr/immutable'
+import { useMemo } from 'react'
+import { ifoV7ABI } from '@pancakeswap/ifos'
+
+import { publicClient } from 'utils/wagmi'
+import { useActiveIfoConfig } from 'hooks/useIfoConfig'
+
+export const useIfoStatus = () => {
+  const activeIfo = useActiveIfoConfig()
+
+  const { data = { startTime: 0, endTime: 0 } } = useSWRImmutable(
+    activeIfo ? ['ifo', 'currentIfo_timestamps', activeIfo.chainId] : null,
+    async () => {
+      const client = publicClient({ chainId: activeIfo.chainId })
```

Comment:
How about, if we only support `BNB` & `ETH` chain for IFO.
But the user is switching to `zkSync` so he will didn't know if any IFO in our supported chains.
Only when the user switches to supported chains then he will know it.